src/main/java/io/cdap/plugin/salesforce/plugin/SalesforceConnectorConfig.java

```diff
@@ -78,19 +79,17 @@ public class BaseSalesforceConfig extends ReferencePluginConfig {
   @Description("Endpoint to authenticate to")
   @Macro
   @Nullable
-  private String loginUrl;
+  public String loginUrl;
 
   private static final String DEFAULT_LOGIN_URL = "https://login.salesforce.com/services/oauth2/token";
```

Comment:
DEFAULT_LOGIN_URL is defined in 3 config classes. Can we dedupe?

Reply:
No, we can't.